fjord/base/email_utils.py

```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+from django.test.client import RequestFactory
+
+
+def render_email(template, context):
+    """Renders a template for email."""
+    req = RequestFactory()
+    req.META = {}
+    resp = render(req, template, context)
+    return resp.content
```

Comment:
OMG! I was looking for that. I have no idea why I didn't find it. I'll fix it with all due haste.